Makefile

```diff
@@ -0,0 +1,9 @@
+temp:	washer.hpp sector.hpp element.hpp BodyModel.hpp PseudoBlockMatrix.hpp SimModel.hpp Simulator.hpp element.cpp BodyModel.cpp Simulator.cpp
+	g++ -o temp.exe Simulator.cpp -g -std=c++11
```

Comment:
What is the purpose of temp? Since there is no main file here, I get a `clang: error: linker command failed with exit code 1 (use -v to see invocation)` error, which could also be the result of something else. My best guess is that it is because there is no main file, and if temp was just for debugging purposes then we should remove it here

Reply:
Correct, I forgot it was even still here. I have been using "g++ test.cpp -o test.exe" then running ./test.exe . Still need to implement some formal tests, but that can wait right now. I had a number of linking errors but rearranged includes to make it work. There should only be 1 include, `#include "Simulator.cpp".